src/Microsoft.ML.FastTree/FastTreeStatic.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Data.StaticPipe.Runtime;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.FastTree;
+using Microsoft.ML.Runtime.Internal.Internallearn;
+using Microsoft.ML.Runtime.Training;
+using System;
+
+namespace Microsoft.ML.Trainers
+{
+    public static class FastTreeStatic
+    {
+        public static Scalar<float> FastTree(this RegressionContext.RegressionTrainers ctx,
+            Scalar<float> label, Vector<float> features, Scalar<float> weights = null,
+            int numLeaves = 20,
```

Comment:
Are you asking why defaults? To give people an idea of a good start, if they wanted to approximate.. 
To avoid dealing with nulls. those are going to be the values if not set to anything. 



---
In reply to: [220018895](https://github.com/dotnet/machinelearning/pull/1009#discussion_r220018895) [](ancestors = 220018895)